### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by jasurganiyevv** [community/community#37489] [Other Feature Feedback, Questions, & Ideas]
How to get quickdraw?

**Comment by DSDmark** [comment]
Hey @jasurganiyevv,

##  To achieve the 'Quickdraw' achievement, you need to:-

> Closed an issue / pull request within 5 minutes of opening

## Here is the list of all badges for future reference:- 

> https://github.com/drknzz/GitHub-Achievements
